spring-boot-starter-rocketmq/.travis.yml

```diff
@@ -0,0 +1,13 @@
+notifications:
+  email:
+    recipients:
+      - angus.aqlu@gmail.com
```

Comment:
Please use dev email list, not personal

Reply:
Where i can get the dev email list?

Reply:
see here, https://github.com/apache/rocketmq/blob/master/.travis.yml